### PR TITLE
feat: add aiops

### PR DESCRIPTION
**Pull request by lifu963**
* 🤖 add release changelog using robot.

* Update CHANGELOG.md

---------

### Summary
:sparkles::memo::whale:

This pull request adds a new AIops service to Sealos, which uses OpenAI to generate chat responses based on the user input and the message history. It also updates the changelog files for the v4.3.4 release. The AIops service consists of a Docker image, a Go module, a main program, and a README file. The `controllers/aiops` directory is added to the `go.work` file.

> _`AIops` service_
> _New files for code and docs_
> _Autumn of Sealos_

### Walkthrough
*  Add a new AIops service that uses OpenAI to generate chat responses based on the message history ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-21efa4a5d7f3d05975e0c405721b6377d30bd8db4a961512d3f23458633e4c65R1-R16), [link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-869b2cc763c99ed2ea04aa22e2c350367493a988f5758fc436ef06aad9bf33fbR1-R45), [link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-c041a96839ccdda960dac8f2ed173c1461be513d4e1a3cd49b2db451c0b01d90R1-R327), [link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-6c004c6d8836fca753b9825f10a93c3423a6cd5b859e829425c5f1206335aac8L1-R234), [link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-70f7e3c361ae136c655130ab253927b5f6e5c987797a095db1bbb1e790930484R26))
  * Define the Docker image for the service in `controllers/aiops/Dockerfile` ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-21efa4a5d7f3d05975e0c405721b6377d30bd8db4a961512d3f23458633e4c65R1-R16))
  * Declare the module name and the dependencies in `controllers/aiops/go.mod` ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-869b2cc763c99ed2ea04aa22e2c350367493a988f5758fc436ef06aad9bf33fbR1-R45))
  * Implement the main logic in `controllers/aiops/main.go` ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-c041a96839ccdda960dac8f2ed173c1461be513d4e1a3cd49b2db451c0b01d90R1-R327))
  * Provide documentation in `controllers/aiops/README.md` ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-6c004c6d8836fca753b9825f10a93c3423a6cd5b859e829425c5f1206335aac8L1-R234))
  * Include the service directory in the workspace in `go.work` ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-70f7e3c361ae136c655130ab253927b5f6e5c987797a095db1bbb1e790930484R26))
* Update the changelog files to document the v4.3.4 release of Sealos ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-ed8afa0a81d92f180e688fc92f4a58ce0eb380322e7d57c854af8a9cf9033954R1-R17), [link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-3f22b1813749a0a5c75b03c440dfbd396f2b251c68251f63822eb492e69d8c87R8))
  * Add a new file `CHANGELOG/CHANGELOG-4.3.4.md` with the release notes ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-ed8afa0a81d92f180e688fc92f4a58ce0eb380322e7d57c854af8a9cf9033954R1-R17))
  * Reference the new file in `CHANGELOG/CHANGELOG.md` under the `Changelog` section ([link](https://github.com/labring/sealos/pull/4021/files?diff=unified&w=0#diff-3f22b1813749a0a5c75b03c440dfbd396f2b251c68251f63822eb492e69d8c87R8))

### CODE REVIEW

**Comment by lifu963** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

File: controllers/aiops/README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+直接复制粘贴文档中的内容：
+
+![image-20230929164037968](images/README/image-20230929164037968.png)
+
+请求：
+
+```sh
+ curl -X POST -H "Content-Type: application/json" -d '{
+  "user_input": "自定义应用名称;镜像名称设置为 halohub/halo:2.5;CPU 和存储应根据实际情况进行配置，memory 需要 500Mi 的内存才能启动 Halo，这里可以参考官方 prepare：https://docs.halo.run/getting-started/prepare;暴露端口应为 8090。同时，设置为外网访问将自动为应用配置一个出口域名用于外网访问，同时需要在 Halo 中进行配置该地址;环境变量包括数据库配置和其他相关配置。需要注意的是，你需要根据你在 步骤 1 中设置的用户名、密码和 DNS 来配置数据库连接信息。同时，你还需要为 Halo 配置超级管理员的用户名和密码。;配置 Halo 环境变量:spring.sql.init.platform=postgresql spring.r2dbc.url=r2dbc:pool:postgresql://postgres:rcx7f47m@dbname-postgresql.ns-sy32q9p9.svc:5432/halo spring.r2dbc.username=postgres spring.r2dbc.password=rcx7f47m halo.external-url=tmtdvfjiyyfh.cloud.sealos.io halo.security.initializer.superadminusername=root halo.security.initializer.superadminpassword=sealos",
```
Comment (fanux) [inline controllers/aiops/README.md:126]: 如果用户这样去用这个产品，那几乎毫无意义，除了增加复杂度，你会发现还不如直接在页面上点表单方便。

**Comment by fanux** [comment]
这个功能如果这样去做没有太大意义，不实用

**Comment by lifu963** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Pull request closed without merging**